packages/stargate/src/stargateclient.ts

```diff
@@ -80,6 +78,29 @@ export class StargateClient {
     }
   }
 
+  /**
+   * Queries all balances for all denoms that belong to this address.
+   *
+   * Uses the grpc queries (which iterates over the store internally), and we cannot get
+   * proofs from such a method.
+   */
+  public async getUnverifiedAllBalances(address: string): Promise<readonly Coin[]> {
+    const path = "/cosmos.bank.Query/AllBalances";
+    const request = QueryAllBalancesRequest.encode({ address: Bech32.decode(address).data }).finish();
+    const responseData = await this.queryUnverified(path, request);
+    const response = QueryAllBalancesResponse.decode(responseData);
+    return (response.balances || []).map(
+      (balance): Coin => {
```

Comment:
This map is quite long and obscured the actual functionality.
Maybe an `assertCoin(any): Coin` function could be added and we just map with that.

Reply:
👍 sure, will pull out (and document and test) the proto Coin to launchpad coin mapping once we need it in a few places.

Reply:
Also suggest a `.forEach` since if an array is returned all the elements are the same as the original array.

Reply:
The idea is to return new element of a different `Coin` type. Input is the protobuf type, outut is the `Coin` interface from @cosmjs/launchpad.